README.md

```diff
@@ -25,15 +36,19 @@ $ cd ../
 $ make build
 ```
 
-To enter the container:
+For serial tests (these take a bit of time), there are two options:
+
+(1) To enter the container and run the dynamical core serial tests (main and savepoint tests):
+
 ```shell
 $ make dev
+$ pytest -v -s --data_path=/fv3core/test_data/8.1.0/c12_6ranks_standard/dycore/ /fv3core/tests
 ```
 
-Then in the container, dynamical core serial tests can be run:
+(2) To run the tests without opening the docker container (just savepoint tests):
```

Comment:
I think you should also specify here that setting `DEV=y` mounts the local directory, so any changes locally will be in effect without having to rebuild the container.

Reply:
Updated